.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Description of what this PR fixes, steps to reproduce.
+
+```
```

Comment:
Won't this mean that if someone wants to use this they would then need to remove the code block comment? Which would require the removal of the beginning/end of the comment tag.